### PR TITLE
NAS-104573 / 11.3 / Fix replication manual snapshot timezone

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/system.py [hunk 1]
```diff
@@ -1067,6 +1068,16 @@ def _get_index(country_columns, column):
 
         raise CallError('Unable to connect to any of the specified UI addresses:\n' + '\n'.join(errors))
 
+    @private
+    async def set_timezone(self):
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/system.py]: Arent't we already doing this in _reload_tinmeservices?

Comment (themylogin) [reply src/middlewared/middlewared/plugins/system.py]: Have not thought about that. Maybe we should not do it there? It's a code duplication.

Comment (william-gr) [reply src/middlewared/middlewared/plugins/system.py]: If you're going to inspect all use cases, I dont mind either way

Comment (themylogin) [reply src/middlewared/middlewared/plugins/system.py]: @william-gr there are no, in TN too (actually that's an issue, who handles timezone change on idle node?)

**Comment by william-gr** [comment]
_reload_timeservices does

service.* calls are replicated to standby

File: src/middlewared/middlewared/plugins/system.py (hunk 1 above)
Comment (william-gr) [reply src/middlewared/middlewared/plugins/system.py]: _reload_timeservices does

service.* calls are replicated to standby

Comment (themylogin) [reply src/middlewared/middlewared/plugins/system.py]: @william-gr then guess I'll stop playing with fire and trying to look smart :)

**Pull request merged**